sklearn/dummy.py

```diff
@@ -67,7 +67,7 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
     classes_ : ndarray of shape (n_classes,) or list thereof
         Class labels for each output.
 
-    n_classes_ : int or list of int
+    n_classes_ : int or list thereof
```

Comment:
We have not reached a consensus in this regard: https://github.com/scikit-learn/scikit-learn/pull/18243#discussion_r475250655